lib/when_committed.rb

```diff
@@ -1,30 +1,42 @@
 require 'when_committed/version'
+require 'active_record'
 
-module WhenCommitted
-  module ActiveRecord
-    def self.included(base)
-      base.after_commit :run_when_committed_callbacks
-      base.after_rollback :clear_when_committed_callbacks
+module WhenCommitted::ActiveRecord
```

Comment:
Thank you @joshuaflanagan, that makes sense.  I'll change it back to be on the safe side.